boost-gradle/src/main/groovy/io/openliberty/boost/liberty/tasks/BoostPackageTask.groovy

```diff
@@ -251,7 +259,12 @@ public class BoostPackageTask extends AbstractBoostTask {
     //returns true if bootJar is using the same archiveName as jar
     private boolean shouldReplaceProjectArchive() {
         if (project.plugins.hasPlugin('java')) {
-            return project.jar.archiveName == project.bootJar.archiveName
+            if (springBootVersion.startsWith('2.')) {
+                return project.jar.archiveName == project.bootJar.archiveName
+            } 
+            // else if (springBootVersion.startsWith('1.')) {
```

Comment:
You need this.

Reply:
Looks like the 1.5 case is always gonna be the jar.archiveName.